tests/angular_devkit/build_angular/hello-world-app/tslint.json

```diff
@@ -130,12 +130,12 @@
       "kebab-case"
     ],
     "no-output-on-prefix": true,
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "no-inputs-metadata-property": true,
```

Comment:
There seems to be more rules to be updated see https://github.com/angular/angular-cli/pull/14007/files